activerecord/lib/active_record/railties/databases.rake

```diff
@@ -261,6 +296,8 @@ db_namespace = namespace :db do
     ActiveRecord::Tasks::DatabaseTasks.for_each(databases) do |name|
       desc "Rollback #{name} database for current environment (specify steps w/ STEP=n)."
       task name => :load_config do
+        raise "VERSION is not supported - To rollback a specific version, use db:migrate:down" if ENV["VERSION"]
```

Comment:
#41430 missed adding the guard for multi DB rollback tasks. 

@eileencodes - one of the drivers for me to remove the namespaced tasks in the initial version was that code was repeated and is easy to be overlooked when enhancing - dummy erb(namespaced tasks not working itself) and this are examples of that. I know that removing wasn't the right way to deal with it, but something to keep in mind for future enhancements.